src/dc_tools.rs

```diff
@@ -675,12 +668,15 @@ pub unsafe fn dc_timestamp_from_date(date_time: *mut mailimf_date_time) -> i64 {
 
 /* the return value must be free()'d */
 pub unsafe fn dc_timestamp_to_str(wanted: i64) -> *mut libc::c_char {
-    let ts = chrono::Utc.timestamp(wanted, 0);
-    let res = ts.format("%Y.%m.%d %H:%M:%S").to_string();
-
+    let res = dc_timestamp_to_str_safe(wanted);
     strdup(to_cstring(res).as_ptr())
 }
 
+pub fn dc_timestamp_to_str_safe(wanted: i64) -> String {
```

Comment:
`dc_timestamp_to_string()`?

Reply:
I wanted safe versions that match the originals 1-1 that's why I kept the name for now